qcodes/instrument_drivers/ZI/ZIHDAWG8.py

```diff
@@ -39,6 +53,19 @@ def __init__(self, name: str, device_id: str, **kwargs) -> None:
         self.awg_module.execute()
         node_tree = self.download_device_node_tree()
         self.create_parameters_from_node_tree(node_tree)
+        self._warnings_as_errors: List[str] = []
+
+    def elevate_compiler_warning_to_error(self, *warnings: str) -> None:
```

Comment:
since there is this "add" functionality, i'd expect also a function for "removing" an already added warning so that it's treated again as a warning, right?)

to make it even easier, instead of creating the "add" and "remove" functions, i'd just make the `_warnings_as_errors` list "public" and write smth in the docs or the example notebook on how to use it.

Reply:
✔️